src/components/molecules/UiHeader/UiHeader.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div class="ui-header">
-    <header class="ui-header__header">
+  <header class="ui-header">
```

Comment:
Q: why do you thing is semantic structure?

Reply:
1. Whole component is header.
2. `ui-header__header` - don't need comment.
3. There are no benefits of current structure, I see it as a mistake. Maye you can write me reasons why it's better and proper?
4. Currently you can't create semantic structure of application (header-main-footer at same deepness):
<img width="425" alt="image" src="https://github.com/infermedica/component-library/assets/20798383/71ae307d-ac29-40b6-833e-6ea12ac7b07e">

Reply:
Why do we have the `header` tag and the `div` inside it as the `__element`? Couldn't we eliminate the `div.ui-header__container` and play it out with a styling (padding, I suppose)? WDYT? :)  What are the pros of having an additional `div.ui-header__container` element in the DOM?

Reply:
It's no reason to these changes IMO. 
* you want <header/><main/><footer/> but IMO it's only one value. Visual value because semantic is still fine. 
* header contains content, this div is the background wrapper, TBH value of header is inside <header/>
* we need this this "wrapper" and content. content is limited and allow to arrange the element on their position with flex

Reply:
Couldn't we work it out by eliminating the `.ui-header__container` element and simply adding styles to the `.ui-header`, like padding inline with the usage of the page-width CSS variable? `padding-inline: calc((100vw - var(--page-width))/2);`

Reply:
I don't agree with you - if it's only "background wrapper" etc. should not be a part of component and that wrapper should be handled on implementation level. If background/spacings/whatever is inseparable part of UiHeader - it's header not a background wrapper.
What do you mean by "header is inside"? Header =/= Navigation/Buttons in it. Going that way further this what you called header content is nav, not header. If this header will be sticked, the sticked will be whole component not "header content" without "background wrapper" like you called it.